girder/test/test_plugin_import.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+import types
+
+
+@pytest.mark.testPlugins(['test_plugin'])
+def testPluginImportSemantics(server):
```

Comment:
Could this be thought of as a test of the `testPlugins` marker itself? I have a local branch where I've been trying to work on testing `pytest-girder`.

Reply:
Yes, it does end up asserting that that decorator is behaving correctly.